scalapb-runtime-grpc/src/main/scala/com/trueaccord/scalapb/grpc/ProtoUtils.scala

```diff
@@ -0,0 +1,25 @@
+package com.trueaccord.scalapb.grpc
+
+import com.google.protobuf.InvalidProtocolBufferException
+import com.trueaccord.scalapb.GeneratedMessage
+import io.grpc.Metadata
+
+object ProtoUtils {
+  def metadataMarshaller[T <: GeneratedMessage](instance: T): Metadata.BinaryMarshaller[T] =
```

Comment:
I mean you create an anonymous class for the marshaller. Maybe (I'm not sure) it's better to use abstract class for the functionality so that it's possible to use that as `object` in other places or override something if the need comes. 
Then the `metadataMarshaller` method would just simply instantiate that.